eth/tracers/internal/tracers/state_diff_tracer.js

```diff
@@ -0,0 +1,537 @@
+			// otherwise account might be re-added for ctx.[from|to|coinbase] from within this.result()
+			if (accountData._error || accountData._removed) {
+				delete this.stateDiff[acc];
+				continue;
+			}
+
+			// check if it is a new borned account
+			// happens on mordor's tx: 0xc468750bd7d73f53ff3fdc74201245910d84d84bfc5c40d97e4c5a8928c92187
+			if (accountData._type === changedMarker
+					&& accountData.balance[memoryMarker].from == 0
+					&& accountData.code[memoryMarker].from === "0x"
+					&& accountData.nonce[memoryMarker].from == 0) {
+				accountData._type = this.diffMarkers.Born;
+			}
+
+			var type = accountData._type;
+			delete accountData._type;
+			delete accountData._removed;
+			delete accountData._error;
+			delete accountData._final;
+
+			if (type === changedMarker) {
+				accountData.balance = this.formatChanged(accountData.balance, type);
+				accountData.nonce = this.formatChanged(accountData.nonce, type);
+				accountData.code = this.formatChanged(accountData.code, type);
+			} else {
+				accountData.balance = this.formatSingle(accountData.balance, type);
+				accountData.nonce = this.formatSingle(accountData.nonce, type);
+				accountData.code = this.formatSingle(accountData.code, type);
+			}
+
+			if (db.empty(toAddress(acc))) {
+				delete this.stateDiff[acc];
+				continue;
+			}
+
+			// optimisation: pre-check if we have changes before parsing storage state
+			if (!this.hasAccountChanges(accountData)) {
+				delete this.stateDiff[acc];
+				continue;
+			}
+
+			// handle storage entries
+			for (var idx in accountData.storage) {
+				// fetch latest value in storage
+				this.lookupStorage(accountAddress, toWord(idx), null, db);
+
+				var sti = accountData.storage[idx];
+				if (sti[memoryMarker] === undefined
+						|| sti[memoryMarker].to === undefined
+						|| !sti[memoryMarker]._changed) {
+					delete this.stateDiff[acc].storage[idx];
+					continue;
+				}
+
+				if (type === changedMarker) {
+					var res = this.formatChanged(sti, type);
+					if (res === this.diffMarkers.Same) {
+						delete this.stateDiff[acc].storage[idx];
+					} else {
+						accountData.storage[idx] = res;
+					}
+				} else {
+					// when Died and from|to is same, then remove the storage entry from output
+					// happens on mordor's tx: 0x0c59ddf8ebbaa64140db6214bbad641fff6bb066847dbef3433d434bd1fb6270
+					if (type === this.diffMarkers.Died && this.checkIfSame(sti, type)) {
+						delete this.stateDiff[acc].storage[idx];
+					} else {
+						accountData.storage[idx] = this.formatSingle(sti, type);
+					}
+				}
+			}
+
+			// remove unchanged accounts
+			if (!this.hasAccountChanges(accountData)) {
+				delete this.stateDiff[acc];
+				continue;
+			}
+		}
+	},
+
+	// includeOpError checks for specific VM OP errors
+	includeOpError: function(err) {
+		return err
+			&& err.indexOf('contract address collision') > -1;
+			// && (err.indexOf('insufficient balance for transfer') > -1
```

Comment:
Dead code or developer clue?

Reply:
It's a developer clue on a flow I found on OE code, but I am not sure if reachable on state_diff and I couldn't find a use case or proof that it is needed.

I will remove it as it doesn't look good. Either way, I have some notes in my agenda with regards such things.

Reply:
Resolved by f553df4